README.md

```diff
@@ -17,53 +17,72 @@
   under the License.
 -->
 
-# Ballista: Distributed SQL Query Engine, built on Apache Arrow
+# Ballista: Making DataFusion Applications Distributed
 
-Ballista is a distributed SQL query engine powered by the Rust implementation of [Apache Arrow][arrow] and
-[Apache Arrow DataFusion][datafusion].
+Ballista is a library which makes [Apache DataFusion](https://github.com/apache/datafusion) applications distributed.
```

Comment:
Ballista provides scheduler and executor processes, so is more than a library?

Reply:
tricky one ... "Ballista is a DISTRIBUTED EXECUTION ENGINE which makes [Apache DataFusion](https://github.com/apache/datafusion) applications distributed." ... i would like to avoid "SQL query engine".

Could not find anything better than that, open to suggestion.